Functions.Templates/Templates/HttpTrigger-PowerShell/run.ps1

```diff
@@ -0,0 +1,24 @@
+# Input bindings are passed in via param block.
+param($req, $TriggerMetadata)
+
+# Write to the Azure Functions log streams as you would in a normal PowerShell script.
```

Comment:
Nit: Write to the Azure Functions log stream.
Remove "as you would in a normal PowerShell script"

Reply:
Updated. Thanks!